components/forms/conditionOfJoining/CojView.tsx

```diff
@@ -11,23 +11,31 @@ import { acceptanceValidation } from "../formr-part-b/ValidationSchema";
 import { Redirect } from "react-router-dom";
 import { updatedsigningCoj } from "../../../redux/slices/userSlice";
 import { COJ_DECLARATIONS } from "../../../utilities/Constants";
+import { DateUtilities } from "../../../utilities/DateUtilities";
 
 const CojView: React.FC = () => {
   const signingCoj = store.getState().user.signingCoj;
   const progName = store.getState().user.signingCojProgName;
+  const signedDate = store.getState().user.signingCojSignedDate;
 
   if (!signingCoj) return <Redirect to="/programmes" />;
   return progName ? (
     <>
       <ScrollTo />
       <CojVersion progName={progName} />
-      <CojDeclarationSection />
+      <CojDeclarationSection canEdit={!signedDate} signedDate={signedDate} />
```

Comment:
I would not have the canEdit prop at all. Just use the signedDate which has Type Date/null (ie. truthy or falsy), so this line would be:

`<CojDeclarationSection signedDate={signedDate} />`

You can then use the signedDate as the conditional in the CojDeclaration section instead of canEdit. 
If you need a boolean type for the MultiChoiceInputField the you can always coerce the signedDate to boolean like this:
`              <MultiChoiceInputField
  ...
                canEdit={!!signedDate}
                checked={!!signedDate}
 ...
                 />`